src/content/getting-started/observability/logging/disable/index.md

```diff
@@ -0,0 +1,76 @@
+---
+linkTitle: Disable logging
+title: Disable Logging
+description: This article explains how to turn off all logging for an entire workload cluster.
+weight: 50
+menu:
+  main:
+    identifier: getting-started-observability-logging-disable
+    parent: getting-started-observability-logging
+user_questions:
+  - How can I disable logging on my cluster?
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-atlas
+last_review_date: 2023-10-11
+aliases:
+  - /ui-api/observability/logging/disable
+  - /observability/logging/disable
+---
+
+In this article you will learn how you can disable logging for your cluster.
+
+## Introduction to logging
+
+Each cluster created on the Giant Swarm platform benefits from our logging stack which allow us to retrieve logs from Kubernetes and GiantSwarm components. 
+Hence we are able to follow components lifecycle and to debug incidents more efficiently.
```

Comment:
Not sure this line should be here. I would probably add it in a Logging section

Reply:
Well, I followed what we did with monitoring disabling...

Reply:
I might have written something like:

Since releases 19.1.0, cluster created on the Giant Swarm platform benefits from our logging stack which allow us to provide you with 24/7 support to ensure best quality of service.